File: app/src/main/java/pl/pregiel/workwork/exceptions/EmptyFieldException.java
package pl.pregiel.workwork.exceptions;


import android.content.Context;
import android.widget.Toast;

import pl.pregiel.workwork.R;

public class EmptyFieldException extends ShowToastException {
    public EmptyFieldException(String message) {
        super(message);
    }

    public void showToast(Context context) {
        Toast.makeText(context, context.getString(R.string.error_emptyField, getMessage()), Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/pl/pregiel/workwork/exceptions/EmptyFieldException.kt
package pl.pregiel.workwork.exceptions


import android.content.Context
import android.widget.Toast

import pl.pregiel.workwork.R

class EmptyFieldException(message: String) : ShowToastException(message) {

    override fun showToast(context: Context) {
        Toast.makeText(context, context.getString(R.string.error_emptyField, message), Toast.LENGTH_SHORT).show()
    }
}
